lib/rubocop/cop/rails/unsafe_query_condition.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks that Active Record queries use literal keys in all of
+      # their conditions. Using dynamic keys in queries can make code
+      # susceptible to SQL injection attacks.
+      #
+      # @example
+      #   # bad
+      #   Model.where("order_count > #{params[:orders]}")
+      #   Model.having("order_count > #{params[:orders]}")
+      #   Model.exists?("order_count > #{params[:orders]}")
+      #
+      #   # good
+      #   Model.where('order_count > ?', params[:orders])
+      #   Model.having('order_count > ?', params[:orders])
+      #   Model.exists?('order_count > ?', params[:orders])
+      #
+      # @example EnforcedStyle: params (default)
+      #   # bad
+      #   Model.find_by(params['id'])
```

Comment:
I'm struggling to understand how this will even work. Will
```ruby
Model.find_by(params['query'])
```
make a better "bad" example?